packages/apollo-link-http-common/src/__tests__/index.ts

```diff
@@ -28,7 +28,7 @@ describe('Common Http functions', () => {
     const operations = [createOperation({}, { query })];
 
     it('throws a parse error with a status code on unparsable response', done => {
-      const status = 400;
```

Comment:
Why does this need changing this should not have effect from the PR

Reply:
Well if status code is 400 I expect it to be `ServerError` instead of `ServerParseError`. The reason is that if status code is not ok I cannot assume the response is valid JSON.

Reply:
I will add an additional unit to cover this case